spec/index.html

```diff
@@ -260,11 +269,17 @@ <h3>RDF Literals</h3>
 
     <p>The corresponding <a data-cite="RDF12-CONCEPTS#dfn-lexical-form">lexical form</a>
       is the characters between the delimiters, after processing any escape sequences.
-      If present, the <a data-cite="RDF12-CONCEPTS#dfn-language-tag">language tag</a>
-      is preceded by an <a href="#cp-at-sign"><code title="at sign">@</code></a>.
+      If present, the <a href="#grammar-production-LANG_DIR" class="type langDir"><code>LANG_DIR</code></a>
+      terminal matches the <a data-cite="RDF12-CONCEPTS#dfn-language-tag">language tag</a>
+      and optionally the <a data-cite="RDF12-CONCEPTS#dfn-base-direction">base direction</a>.
+      The <a data-cite="RDF12-CONCEPTS#dfn-language-tag">language tag</a>
+      is preceded by an <a href="#cp-at-sign"><code title="at sign">@</code></a>,
+      and, if present, the <a data-cite="RDF12-CONCEPTS#dfn-base-direction">base direction</a>
+      is separated from the <a data-cite="RDF12-CONCEPTS#dfn-language-tag">language tag</a>
+      by <a href="#cp-hyphen-hyphen"><code>--</code></a>.
```

Comment:
The `--` base direction separator might be better expressed and presented as was done for the `^^` type separator, as some rendering paths will "helpfully" change the `--` hyphen-hyphen to `—` m-dash or `–` n-dash or possibly some other dash.

Reply:
See the current handling in [rdf-n-triples](https://github.com/w3c/rdf-n-triples/blob/main/spec/index.html#L269-L272) for what I was suggesting and referencing —
```
        by <code>--</code> (two concatenated hyphen characters, each having the code point <code class="codepoint">U+002D</code>).
        If there is no language tag, there may be a <a data-cite="RDF12-CONCEPTS#dfn-datatype-iri">datatype IRI</a>,
        preceded by <code>^^</code>
        (two concatenated circumflex accent characters, each having the code point <code class="codepoint">U+005E</code>).
```

I have encountered the rendering transformation of two concatenated hyphens to unintended mdash or ndash characters in various places, hence my consistent concern with presentation of two concatenated hyphens. I have never encountered any rendering transformation of two concatenated circumflex to any unintended character or sequence of characters, so my concern with presenting two concatenated circumflex is just making such roughly parallel to the concatenated hyphens, for easier overall comprehension.

Reply:
The referenced rdf-n-triples document does not include changes from https://github.com/w3c/rdf-n-triples/pull/46, which uses the same mechanism we're using elsewhere for referring to grammar tokens such as `--`. We've used that consistently for single- and multiple-character sequences such as `--`, `^^`, `<<`, `>>`, and so forth. By making these links into the **Selected Terminal Literal Strings** section, we remove the ambiguity.

Reply:
OK. You can mark this one resolved. (I would, but I have no such button.)